week-4/B-objects-get-set/exercise-4.js

```diff
@@ -3,21 +3,22 @@
 */
 
 var dog = {
-    name: 'Billy',
-    wantsToPlay: false
+  name: "Billy",
+  wantsToPlay: false
 };
 
 // WRITE CODE BELOW THIS LINE
-
-
+var dog = {

```

Comment:
The idea here is to modify the `dog` object, not to create a new one, shadowing the original.